02_configure_host.sh

```diff
@@ -212,3 +212,11 @@ else
     # Create a blank authfile in order to have something valid when we read it in 04_setup_ironic.sh
     echo '{}' | sudo dd of=${REGISTRY_CREDS}
 fi
+
+# metal3-dev-env contains a script to run the openstack client in a
+# container, place a link to it in $PATH if we don't already have the
+# openstack command installed
+OPENSTACKCLIENT_PATH="${OPENSTACKCLIENT_PATH:-/usr/local/bin/openstack}"
+if ! command -v openstack | grep -v "${OPENSTACKCLIENT_PATH}"; then
+	  sudo ln -sf "${WORKING_DIR}/metal3-dev-env//openstackclient.sh" "${OPENSTACKCLIENT_PATH}"
+fi
```

Comment:
nit: double slashes here `//`

Reply:
Also there's a corner case where this will use the wrong metal3-dev-env if someone sets METAL3_DEV_ENV to a local checkout:

https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh#L188

We could set METAL3_DEV_ENV_PATH there similar to VM_SETUP_PATH perhaps and use that here instead?

Reply:
will do